framework/include/materials/ParsedMaterialHelper.h

```diff
@@ -83,6 +83,7 @@ class ParsedMaterialHelper : public FunctionMaterialBase<is_ad>, public Function
 
   void initQpStatefulProperties() override;
   void computeQpProperties() override;
+  void initialSetup() override final;
```

Comment:
Even though `override` implies `virtual`, I've been adding `virtual` in front lately since it helps when reading the resulting doxygen

Reply:
Done.